Composer/packages/extensions/visual-designer/src/components/nodes/templates/FormCard.tsx

```diff
@@ -124,12 +125,13 @@ export const FormCard: FunctionComponent<NodeProps> = ({
             css={{
               height: '100%',
               width: 'calc(100% - 20px)',
-              whiteSpace: 'nowrap',
-              textOverflow: 'ellipsis',
-              overflow: 'hidden',
+              whiteSpace: 'initial',
               fontSize: '12px',
               lineHeight: '19px',
               fontFamily: 'Segoe UI',
+              overflowWrap: 'break-word',
+              wordBreak: 'break-all',
+              display: 'contents',
```

Comment:
Can you make sure this has the browser support necessary? https://developer.mozilla.org/en-US/docs/Web/CSS/display

Reply:
I changed it to 'inline-block' and confirmed no other usage of `display: contents`